src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk

```diff
@@ -0,0 +1,116 @@
+  </p>
+{% endmacro %}
+
+{% macro claimRejectedAlreadyPaidForDefendantDashboard(claim) %}
+  {{ t('We’ve emailed {{ claimantName }} telling them when and how you paid the claim.',
+    { claimantName: claim.claimData.claimant.name }) }}
+{% endmacro %}
+
+{% macro claimRejectedAlreadyPaidForDefendantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('Your response to the claim') }}</h2>
+  <p>
+    {{ t('We’ve emailed {{ claimantName }} telling them when and how you said you paid the claim.',
+      { claimantName: claim.claimData.claimant.name }) }}
+  </p>
+  <p>
+    {{ t('We’ll contact you to let you know how they respond. They can confirm you’ve paid and the claim is ' +
+      'settled, or they can proceed with it.') }}
+  </p>
+  <p>
+    {{ _downloadResponseLink(claim, 'Download your response') }}
+  </p>
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationNOForClaimantDashboard(claim) %}
+  {{ t('{{ defendantName }} has rejected your claim.',
+    { defendantName: claim.claimData.defendant.name }) }}
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationNOForClaimantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('The defendant’s response') }}</h2>
+  <p>
+    {{ t('{{ defendantName }} has rejected your claim. We’ll send you a directions questionnaire – '
+      + 'this is a form which you must complete to tell us more about the claim.',
+      { defendantName: claim.claimData.defendant.name }) }}
+  </p>
+  <p>
+    {{ _downloadResponseLink(claim, 'Download their response') }}
+  </p>
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationNOForDefendantDashboard() %}
+  {{ t('You’ve rejected the claim.') }}
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationNOForDefendantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('Your response to the claim') }}</h2>
+  <p>
+    {{ t('You have rejected the claim. We’ll send you a directions questionnaire – this is a form which you must '
+      + 'complete to tell us more about the claim.') }}
+  </p>
+  <p>
+    {{ _downloadResponseLink(claim, 'Download your response') }}
+  </p>
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationYESForClaimantDashboard(claim) %}
+  {{ t('{{ defendantName }} has rejected the claim. But they’ve suggested a mediation session to help resolve this dispute.',
+    { defendantName: claim.claimData.defendant.name }) }}
+{% endmacro %}
+
+{% macro claimRejectedFreeMediationYESForClaimantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('The defendant’s response') }}</h2>
+  <p>
+    {{ t('{{ defendantName }} has rejected the claim. They’ve suggested mediation to help resolve this dispute.',
+      { defendantName: claim.claimData.defendant.name }) }}
+  </p>
+  <p>
+    {{ t('Tell us whether you agree to mediation or not by emailing {{ emailAddress }} before {{ respondToMediationDeadline }}.',
+      { emailAddress: 'moneyclaims@hmcts.gsi.gov.uk', respondToMediationDeadline: claim.respondToMediationDeadline | date }) }}
```

Comment:
this email address should be pulled out to a configurable variable

Reply:
Good spot. It crossed my mind at some point but then I forgot about it. Thanks for another review.